collector/ad.go

```diff
@@ -612,7 +612,7 @@ type Win32_PerfRawData_DirectoryServices_DirectoryServices struct {
 
 func (c *ADCollector) collect(ch chan<- prometheus.Metric) (*prometheus.Desc, error) {
 	var dst []Win32_PerfRawData_DirectoryServices_DirectoryServices
-	q := wmi.CreateQuery(&dst, "")
+	q := queryAll(&dst)
```

Comment:
for the sake of consistency, could this line be changed in the same way so they look as in eg collector/cs.go ?

Reply:
It actually looks like that we should, for consistency, change cs.go and the few others who follow that pattern (19 like this vs 4 inlined). But I agree, should be the same everywhere.

Reply:
I would do the 1-line version myself but its more edits. Your call :-)